src/quick_start/browser_extension.md

```diff
@@ -0,0 +1,113 @@
+# TLSNotary Browser Extension <a name="browser"></a>
+
+In this Quick Start we will prove ownership of a Twitter account with TLSNotary's browser extension.
+First we need to [install](#instal) and configure a [websocket proxy](#proxy) and a [notary server](#notary-server).
+
+## Install Browser Extension (Chrome/Brave) <a name="install"></a>
+
+1. Download the browser extension from <https://github.com/tlsnotary/tlsn-extension/releases/download/0.1.0.3/tlsn-extension-0.1.0.3.zip>
```

Comment:
Do you plan to update this doc whenever a new version is released?

Reply:
@yuroitaki For now, yes.

I did not find a way to have a link to the latest version. There will probably be other changes too for every release. We'll see

Reply:
Got it, I was thinking to link to the release page and just ask people to use the latest version, but this works too :)